docs/components/DisablingNodes.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <treeselect
+    :multiple="true"
+    :options="options"
+    :show-count="true"
+    :allways-opened="true"
+    placeholder="Select your favourite(s)..."
+  />
+</template>
+
+<script>
+  export default {
+    data: () => ({
+      options: [ {
+        id: 'fruits',
+        label: 'Fruits',
+        children: [ {
+          id: 'apple',
+          label: 'Apple 🍎',
+        }, {
+          id: 'appletwo',
+          label: 'Apple Two 📺',
+          _disabled: true,
```

Comment:
I think `disabled` without a `_` should be good here.

Reply:
I prefer the prefix `_` to distinguish between business attributes and components logic attribute. But writing this I'm seeing that `id` and `label` are components logic attribute so ... `disabled` is good ! 👍